staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5173,6 +5194,14 @@ type SecurityContext struct {
 	// PodSecurityContext, the value specified in SecurityContext takes precedence.
 	// +optional
 	RunAsNonRoot *bool `json:"runAsNonRoot,omitempty" protobuf:"varint,5,opt,name=runAsNonRoot"`
+	// Indicates that the container must run as a non-root group.
+	// If true, the Kubelet will validate the image at runtime to ensure that it
+	// does not run as GID 0 (root) and fail to start the container if it does.
+	// If unset or false, no such validation will be performed.
+	// May also be set in SecurityContext.  If set in both SecurityContext and
+	// PodSecurityContext, the value specified in SecurityContext takes precedence.
+	// +optional
+	RunAsNonRootGroup *bool `json:"runAsNonRootGroup,omitempty" protobuf:"varint,9,opt,name=runAsNonRootGroup"`
```

Comment:
Is this actually read anywhere?

Reply:
no, this is not implemented, only the RunAsGroup field is. there is a feature gate to disable it